scripts/test-satellite-cfg-change.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -
+
+# NOTE this script MUST BE EXECUTED from the same directory where it's
+# located to obtain the same paths generated by the satellite binary
+
+set -ueo pipefail
+
+TESTDATA_DIR="./testdata"
+
+#setup tmpdir for testfiles and cleanup
+#TMP_DIR=$(mktemp -d -t satellite-cfg-change-detector-XXXXX)
+cleanup(){
+  rm "$TESTDATA_DIR/config.yaml"
+}
+trap cleanup EXIT
+
+satellite --config-dir "$TESTDATA_DIR" --defaults release setup > /dev/null
+
+
+diff "$TESTDATA_DIR/satellite-config.yaml.lock" "$TESTDATA_DIR/config.yaml"
+if [[ $? != 0 ]]; then
+  echo "Notify the Devops and PM when this test fails so they can plan for changing it in the release process before fixing it to merge your PR"
```

Comment:
Can you include directions on how to update 'satellite-config.yaml.lock' here as well?

Reply:
(Possibly adding a makefile task to make things easy)

Reply:
Added a script and Makefile target to update the config lock file.